pkgs/top-level/stage.nix

```diff
@@ -49,6 +49,13 @@
   # be defined internally as the produced package set as itself.
   buildPackages
 
+, # The package set used in the next stage. If null, `__targetPackages` will be
+  # defined internally as the produced package set as itself.
+  #
+  # THIS IS A HACK for compilers that don't think critically about cross-
+  # compilation. Please do *not* use unless you really know what you are doing.
```

Comment:
Just curious, but what compilers need __targetPackages? It looks like from your other commits that it is only used by glibc?